src/test/kotlin/interactor/GetCitiesHasTheCheapestBananaPricesTest.kt

```diff
@@ -0,0 +1,123 @@
+package interactor
+
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import model.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.function.Executable
+
+class GetCitiesHasTheCheapestBananaPricesTest{
+
+    @MockK
```

Comment:
in your tests, you're re-initializing your interactor class. so its never been used as a mock object.
remove the @MokK annotation and just leave it as a lateinit var.